src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
1) Don't use verbs for class/interface names
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-verbs-for-classinterface-names
2) Seem like you don't use this interface anywhere, so instead of abstract method in class Figure just implement this interface